public/index.html

```diff
@@ -47,6 +47,7 @@
       >You need to enable JavaScript to run this
       app.</noscript
     >
+    <div id="modal"></div>
```

Comment:
why do you place the modal here? you could have placed it inside the app itself

Reply:
because i have created portal to render `modals` so there are two ways i can do this 
1. by rendering on body
2. creating new `div` and `id` (my approach)